### DISCUSSION TITLE
How to add tag in YAML

### DISCUSSION THREAD

**Discussion by nousernameavailableanymore** [RafaelGB/obsidian-db-folder#563] [Q&A]
### What happened?

I m using "Tag" as "Select the source of database data" and provided the tag to look for.

Every time I create a new note, this tag gets added as an inline field instead of a YAML field which looks super ugly.

How can I have the tag set as a YAML field instead of inline field?

### What platform were you using?

Desktop

### Version of the plugin

2.7.1

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment]
Hi @nousernameavailableanymore ,

You can select the behaviour of your column inside the column settings. Disable inline field and edit the cell again

Check our documentation
https://rafaelgb.github.io/obsidian-db-folder/features/Properties/#yaml-dataview-inline-fields

**Reply by nousernameavailableanymore** [reply]
There is no column for the "select the source of database data".
This field is only to select which notes are shown in the database. 
So as there is no column to define this tag setting.

![image](https://user-images.githubusercontent.com/45376324/198901735-051c7cf5-0ceb-47b6-9250-552fdd9ea2be.png)

**Reply by nousernameavailableanymore** [reply]
@RafaelGB 
See above response.
This is not what I m looking for.
1. this is the tag to identify, which notes are shown in the table. This is NOT a column within the table. There is **no column** "hasi_log".
2. Inline fields is already disabled (but as explained about, this is not relevant here anyways as this setting refers to the columns in the table)
![image](https://user-images.githubusercontent.com/45376324/200101567-d0657ee2-1129-4c57-9d75-485f593536e0.png)

**Reply by RafaelGB** [reply]
I got it now, sorry.

The create row function (https://github.com/RafaelGB/obsidian-db-folder/blob/master/src/services/FileManagerService.ts) add the tag   at the start of the file.

Can't be modified yet.

We can dev an option about how to persist it:
- yaml
- top of note
